### DISCUSSION TITLE
Confused on why theme is missing number of changes

### DISCUSSION THREAD

**Discussion by dancasmassonos** [JanDeDobbeleer/oh-my-posh#987] [Troubleshoot] [answered]
On one of my machines using the same theme I get the number of changes.  I copied that theme to a new computer and the number of changes is missing. I tried different themes as well and for some reason it seems one computer does not show the number of changes.
Same fonts installed on both machines and nothing obviously different.

Thoughts ?

**Comment by JanDeDobbeleer** [comment]
I'm going to need some more information to be able to help. Can you share some screenshots/settings?

**Reply by dancasmassonos** [reply]
on my home computer the 2 in the photo is also in the bottom line. (BTW thank you for the product!)
![missing 2](https://user-images.githubusercontent.com/91165638/134571939-0c2f4214-c74f-4c5a-91b3-f0efb5aad3d1.jpg)

**Reply by dancasmassonos** [reply]
The number in the main area of the photo doesn't appear like it does on my other computer.

**Reply by JanDeDobbeleer** [reply]
Did you enable git status? Could be you have an old version on computer 1 and the new version on computer 2 which has this disabled by default. See the [git segment info](https://ohmyposh.dev/docs/git) for more details.

**Comment by dancasmassonos** [comment, marked as answer]
That looks like it might be it. Sorry I missed that.

**Reply by JanDeDobbeleer** [reply]
Happy to see that resolved 👍🏻
